src/main/resources/plugin.yml

```diff
@@ -2,7 +2,7 @@ name: CrazyCrates
 main: me.badbones69.crazycrates.Main
 author: BadBones69
 version: ${project.version}
-api-version: 1.13
+api-version: 1.14
```

Comment:
This breaks 1.13 compatibility

Reply:
Yes it does, I was asked to update this plugin from a discord friend who was in a bind, and I did not do so with the intention of it being able to run on 1.13, feel free to close the PR. I do not plan on updating it.